givenergy_modbus/model/register.py

```
@@ -112,16 +157,36 @@ class RegisterDefinition:
     pre_conv: Callable | tuple | None
     post_conv: Callable | tuple[Callable, Any] | None
     registers: tuple["Register"]
+    min: int | float | None
+    max: int | float | None
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, min: int | float | None = None, max: int | float | None = None):
```

Ruff (0.15.12)

[error] Function argument `min` is shadowing a Python builtin

(A002)